app/services/users/delete_account_service.rb

```diff
@@ -73,7 +73,9 @@ def create_audit_record
         metadata: {
           email: @user.email,
           account_deletion_notification_sent_at:
```

Comment:
**Unrelated change:** Please also audit the name of the user who is deleting their account.